lib/tapioca/runtime/trackers/constant_definition.rb

```diff
@@ -51,7 +51,11 @@ class ConstantLocation < T::Struct
 
         def self.build_constant_location(tp, locations)
           file = resolve_loc(caller_locations)
-          lineno = file == File.realpath(tp.path) ? tp.lineno : 0
+          lineno = 0
+
+          if File.exist?(tp.path) && file == File.realpath(tp.path)
```

Comment:
We may want to check `tp.path` as well because it can also be `nil`. We had this issue in the debugger before: https://github.com/ruby/debug/issues/466

It's a very rare case though